### DISCUSSION TITLE
D&D 앱 ?

### DISCUSSION THREAD

**Discussion by sgd122** [DNDACADEMY/dnd-notice#7] [Ideas]
각 사용자들의 모임구성원들에 대해서
- 각 구성원별 Repo 확인 가능
- Repoe에 대한 다양한 기능 제공
- 어떤 구성원이 가장 활동이 활발한지등에 대해서 알 수 있다!

**Comment by rura6502** [comment]
실용적인 기능들이나 기간이 끝나더라도 기수간이나 외부인들과의 커뮤니케이션 기능이 있으면 괜찮을것같습니다.
